docs/en_US/guide/sinks/builtin/mqtt.md

```diff
@@ -14,6 +14,9 @@ The action is used for publish output message into an MQTT server.
 | certificationPath    | true     | The certification path. It can be an absolute path, or a relative path. If it is an relative path, then the base path is where you excuting the `kuiperd` command. For example, if you run `bin/kuiperd` from `/var/kuiper`, then the base path is `/var/kuiper`; If you run `./kuiperd` from `/var/kuiper/bin`, then the base path is `/var/kuiper/bin`. |
 | privateKeyPath       | true     | The private key path. It can be either absolute path, or relative path, which is similar to use of certificationPath.                                                                                                                                                                                                                                     |
 | rootCaPath           | true     | The location of root ca path. It can be an absolute path, or a relative path, which is similar to use of certificationPath.                                                                                                                                                                                                                               |
+| rawCert            | true | base64 encoded original text of cert |
```

Comment:
Two suggestions:
1. Name the property to be a counterpart of the original property. certificationRaw vs certificationPath; privateKeyRaw vs privateKeyPath, rootCaRaw vs. rootCaPath. This can help people easily find out the relationship between them.
2. Demonstrate the priority of each pair, xxPath takes precedence over xxRaw.